hera_cal/chunker.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright 2021 the HERA Project
+# Licensed under the MIT License
+
+
+from . import io
+import argparse
+import numpy as np
+import sys
+import warnings
+
+
+def chunk_data_files(filenames, inputfile, outputfile, chunk_size,
+                     polarizations=None, spw_range=None, throw_away_flagged_bls=False,
+                     clobber=False, ant_flag_yaml=None):
+    """A data file chunker
```

Comment:
Kind of a nit-pick, but the first line of a docstring should be written as a command. So in this case, something like "Chunk a list of data files together into a single file." See [the documentation on PEP257 for more info.](https://www.python.org/dev/peps/pep-0257/#id16)

Reply:
This is a command now. Thanks for the info :)